lib/integration_helpers/deeply_nested_payload_generator.rb

```diff
@@ -153,9 +153,4 @@ def store_client_id(row)
     _object, _source, _attr, value = row
     @payment_hash[client_id_method] = value
   end
-
-  # TODO: Remove #create_client_id method when integration tests spreadsheet passess in client_id with payments
-  def create_client_id(_row)
-    @payment_hash[client_id_method] = FAKE_CLIENT_ID
```

Comment:
Looks good. Just a little leftover code. The point where this constant is defined "FAKE_CLIENT_ID" needs to be removed. On line 31 of this file.

Reply:
Thanks. Fixed!